packages/content-conformance/src/rules/no-link-in-heading.js

```diff
@@ -0,0 +1,41 @@
+import { visit } from 'unist-util-visit'
+
+/**
+ * Visit each node in a subtree to build a string representation
+ */
+function getStringValue(node) {
```

Comment:
💭 Very optional: I think we could use [mdast-util-to-string](https://github.com/syntax-tree/mdast-util-to-string) here, if we want! I tried it out recently in https://github.com/hashicorp/dev-portal/pull/1452 and it seemed to work as expected 👍